### DISCUSSION TITLE
Cucumber BDD implementation with this framework

### DISCUSSION THREAD

**Discussion by sarik12** [akshayp7/playwright-typescript-playwright-test#41] [Ideas]
Is it possible to implement cucumber BDD with this framework?

**Comment by akshayp7** [comment]
Currently I am trying out with Cucumber js. It might take some time as I need to look into few things. 
In mean time if you want to check out playwright cucumber please visit this

https://medium.com/@manabie/how-to-use-playwright-in-cucumberjs-f8ee5b89bccc
